package.json

```diff
@@ -360,10 +362,44 @@
     "test-visual": "yarn build && ./bin/build-for-test && yarn test-visual-run",
     "test-visual-open": "percy exec -- yarn test-cypress-open",
     "test-visual-run": "percy exec -- yarn test-cypress-run --spec \"./e2e/test/visual/**/*.cy.spec.js\"",
-    "type-check": "yarn clean:cljs && yarn build:cljs && yarn type-check-pure",
+    "type-check": "wireit",
     "type-check-pure": "tsc --noEmit",
     "validate-e2e-filenames": "node e2e/validate-e2e-test-files.js",
-    "wait:cljs": "echo Waiting for first CLJS build; until [ -f target/cljs_dev/cljs.core.js ]; do sleep 1; done; echo CLJS build done"
+    "wait:cljs": "echo Waiting for first CLJS build; until [ -f target/cljs_dev/cljs.core.js ]; do sleep 1; done; echo CLJS build done",
+    "build:drivers": "wireit"
+  },
+  "wireit": {
+    "test-cypress-open": {
+      "command": "./bin/build-for-test && yarn test-cypress-run --e2e --open",
+      "dependencies": [
+        "build:drivers"
+      ]
+    },
+    "build:drivers": {
+      "command": "bash ./bin/build-drivers.sh",
+      "files": [
+        "modules/drivers/",
+        "!modules/drivers/*/target"
+      ],
+      "output": [
+        "resources/modules"
+      ]
+    },
+    "type-check": {
```

Comment:
i tried it with type-check too, time wise the savings weren't that big as build:cljs is probably very fast